src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -146,11 +146,7 @@ public Task<IEnumerable<TResult>> ReadAsync<TResult>(string query, Action<IDbCom
                 throw new ArgumentNullException("selector");
```

Comment:
By the way, even if it should be really small chance to generate the same ID randomly, there's such a chance. Originally the idea was to test also whatver strange IDs one has from a given character set, just in case to see there isn't problems in there. One way forwards would to just take, say, two of that just numerated set, copy the third one from either one and change a few latters. This way the test set is always generic.